postgresql-17.yaml

```diff
@@ -195,6 +197,23 @@ subpackages:
       provides:
         - libpq=${{package.full-version}}
 
+  - name: plpython3
+    description: PostgreSQL Python libraries
+    dependencies:
+      provider-priority: 17
+      provides:
+        - plpython=${{package.full-version}}
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/lib/postgresql${{vars.major-version}}
+          mkdir -p ${{targets.subpkgdir}}/usr/share/postgresql${{vars.major-version}}/extension
+      - runs: |
+          mv ${{targets.destdir}}/usr/share/postgresql${{vars.major-version}}/extension/plpython* ${{targets.subpkgdir}}/usr/share/postgresql${{vars.major-version}}/extension/
+          mv ${{targets.destdir}}/usr/lib/postgresql${{vars.major-version}}/plpython* ${{targets.subpkgdir}}/usr/lib/postgresql${{vars.major-version}}/
```

Comment:
note:  we're working on adding `shellcheck` support as a linter, and I think this will cause a shellcheck error and thus a failure when that lands. It's recommended alternative is to mv `$(echo ${{targets.destdir}}/usr/lib/postgresql${{vars.major-version}}/plpython*)` so that the shell will expand the glob.